server/integration-test/src/test/java/com/oceanbase/odc/server/web/controller/v2/WorksheetControllerIntegrationTest.java

```diff
@@ -323,6 +324,43 @@ public void listWorksheets() throws Exception {
                         Collectors.toList()));
     }
 
+    @Test
+    public void flatListWorksheets() throws Exception {
```

Comment:
complex unit testing case here, often we don't write logic in Controller class, so ODC does not calculate Unit Testing coverage for Controller classes

Reply:
Okay，I'll add WorksheetServiceFacadeImplTest to WorksheetControllerIntegrationTest